scenarios/tracking/README.md

```diff
@@ -2,4 +2,131 @@
+
+<p align="center">
+<img src="./media/table_trackerperformance.jpg" width="400" align="center"/>
+</p>
+
+### Evaluation metrics
+As multi-object-tracking is a complex CV task, there exists many different metrics to evaluate the tracking performance.  Based on how they are computed, metrics can be event-based [CLEARMOT metrics](https://link.springer.com/content/pdf/10.1155/2008/246309.pdf) or [id-based metrics](https://arxiv.org/pdf/1609.01775.pdf). The main metrics used to gauge performance in the [MOT benchmarking challenge](https://motchallenge.net/results/MOT16/) include MOTA, IDF1, and ID-switch.
+* MOTA (Multiple Object Tracking Accuracy): it gauges overall accuracy performance, with event-based computation of how often mismatch occurs between the tracking results and ground-truth. MOTA contains the counts of FP (false-positive), FN(false negative) and id-switches (IDSW), normalized over the total number of ground-truth (GT) tracks.
+<p align="center">
+<img src="./media/eqn_mota.jpg" width="200" align="center"/>
+</p>
+
+* IDF1: gauges overall performance, with id-based computation of how long the tracker correctly identifies the target. It is the harmonic mean of identification precision (IDP) and recall (IDR): 
+<p align="center">
+<img src="./media/eqn_idf1.jpg" width="400" align="center"/>
+</p>
+
+* ID-switch: when the tracker incorrectly changes the ID of the trajectory. This is illustrated in the following figure: in the left box, person A and person B overlap and are not detected and tracked in frames 4-5, resulting in an id-switch in frame 6, where person A is attributed the ID_2, which previously tagged person B. In another example in the right box, the tracker lost track of person A (initially identified as ID_1) after frame 3, and eventually identified that person with a new ID (ID_2) in frame n, showing another instance of id-switch. 
+
+<p align="center">
+<img src="./media/fig_tracksEval.jpg" width="600" align="center"/>
+</p>
+
+
+### Popular datasets
+<p align="center">
+<img src="./media/table_datasets.jpg" width="900" align="center"/>
+</p>
+
+### Popular publications
+* General overview #TODO
+* Benchmarking #TODO
+* Seminal/baseline method papers #TODO
+* Recent online & offline trackers, see table below #TODO: publication lists + deep-dive comments, check with @Liqun
+
+<p align="center">
+<img src="./media/table_onlinepub.jpg" width="900" align="center"/>
```

Comment:
How about here we list just the papers we short-listed (ie some select papers) and provide some information such as 
Name, Year, IDF1, MOTA, Online/Batch. Maybe also some of the other columns (detector, association approach, etc) but would have to be short + precise + correct.